source/standards/programming-languages.html.md.erb

```diff
@@ -101,6 +101,30 @@ We used Scala in the early days of GDS. GOV.UK Licensing is the only remaining
 application written in Scala but we've found it hard to support because of a lack
 of skills in GDS. Do not use Scala for new projects.
 
+## Mobile Development
+
+For developing mobile apps, we use:
+- [Swift](https://www.swift.org/) for [iOS](https://developer.apple.com/)
+- [Kotlin](https://developer.android.com/kotlin) for [Android](https://developer.android.com/).
+
+To give users the expected experience on their respective platform, we prefer to use the native languages over cross-platform solutions. 
+
+### Swift
+
+When starting a new app project, you'll likely want to use the most recent version of Swift which will be installed with Xcode.
+
+Deciding on which iOS version you want to support is a more important version and is a tradeoff between making the app accessible to as many devices as possible without compromising security.
+
+The iOS target should also be considered - for government apps, a range of n - 5 or n - 3 is generally a good starting point (n being the latest major release of iOS)
```

Comment:
Thanks, good things to point out! I've tried to make it more clear for both Android and iOS, but as far as I know something like the browser support doesn't yet exist for mobile and is probably something mobile devs across GDS should sit down and create so we can add a useful page like we have for the web

Reply:
Could you change "afford" to "access".  Suggest also adding back the version ranges as a specific example of a policy if that is what your area is using.  That will be a good pointer for people to ask about it.

It would be good to have a browser support-like page but it shouldn't block this page being added as it can be updated later

Reply:
Good catch, I've updated it now and also came across a link Google put advising on the required minimum Android version for apps to go on the Play Store. 

Definitely something to consider for the future, I'd like to think this little section is just the beginning to adding more mobile-orientated advice to the GDS Way!